src/Serilog.Sinks.MongoDB/Sinks/MongoDB/MongoDBSink.cs

```diff
@@ -82,6 +82,16 @@ public MongoDBSink(IMongoDatabase database, int batchPostingLimit, TimeSpan peri
             _collectionName = collectionName;
             _collectionCreationOptions = collectionCreationOptions;
             _formatProvider = formatProvider;
+
+            //  capped collections have to be created because GetCollection doesn't offer the option to create one implicitly
+            //  only create one if it doesn't already exist
+            if (_collectionCreationOptions.Capped.GetValueOrDefault(false)) {
```

Comment:
Would be nice to put `{`s on new lines to match the coding style in the rest of this file.